packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -195,6 +196,25 @@ const DataGrid = forwardRef((props, ref) => {
   } = props;
 
   const displayedColumns = (hasSelectableRows ? [WorklistDataGridUtils.ROW_SELECTION_COLUMN] : []).concat(pinnedColumns).concat(overflowColumns);
+  // Create new displayedColumns object to account for column spans
+  const displayedColumnsWithColumnSpan = useMemo(() => {
```

Comment:
Do we need both of these objects?  It seems like we should be able to drive off one.

Reply:
Discussed offline. We do need these two objects.